webapp/store/rootReducer.js

```diff
@@ -36,12 +36,16 @@ export default function getPersistedReducer() {
     if (action.type === RESET_STATE) {
       // need to clear local storage as well so it's not persisted
       Object.keys(state).forEach(key => {
-        storage.removeItem(`persist:${key}`);
+        if (key !== 'plugins') storage.removeItem(`persist:${key}`);
       });
+
       state.chain = undefined;
       state.wallets = undefined;
       state.node = undefined;
-      state.plugins = undefined;
+      // after this clearing below, it's no longer in the local storage in browser
+      Object.keys(state.plugins).forEach(key => {
+        if (key !== '_persist') state.plugins[key] = undefined;
```

Comment:
so this deletes anything that isn't in the `_persist` field?

Reply:
correct. It actually just skips it because redux doesn't actually recognize the key since it's added separately by the persister. Not skipping it would result in an error in the console from redux (despite the error message though I don't think it actually messed with functionality).